scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
I'm not sure I'd count 7-zip as "sketchy" ? :-S

@WasabiFan The Windows built-in zip has problems with files bigger than 4GB (which happens quite often with disk images), so if you want to manually extract those you're _forced_ to use something like 7-zip.

@jviotti From a technical point of view, you've listed 7-zip as a runtime dependency, and I guess strictly speaking it's only a build / packaging dependency.